driver/mounter.go

```diff
@@ -208,6 +232,26 @@ func (m *mounter) Unmount(target string) error {
 	return mount.CleanupMountPoint(target, m.kMounter, true)
 }
 
+func (m *mounter) IsAttached(source string) (bool, error) {
+	out, err := m.attachmentValidator.evalSymlinks(source)
+	if err != nil {
+		return false, fmt.Errorf("error evaluating the symbolic link %q: %s", source, err)
+	}
+
+	_, deviceName := filepath.Split(out)
+	deviceStateFilePath := fmt.Sprintf("/sys/class/block/%s/device/state", deviceName)
+	deviceStateFileContent, err := m.attachmentValidator.readFile(deviceStateFilePath)
+	if err != nil {
+		return false, fmt.Errorf("error reading the device state file %q: %s", deviceStateFilePath, err)
+	}
+
+	if string(deviceStateFileContent) != fmt.Sprintf("%s\n", runningState) {
```

Comment:
This could probably be a tad more robust if we used [`strings.TrimSpace()`](https://pkg.go.dev/strings#TrimSpace), allowing us to not care what the newline character implementation is.

Reply:
Your proposition is cleaner indeed. Adding it.